src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,139 @@
+package storage;
+
+import exception.TobiasException;
+import parser.Parser;
+import task.*;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+
+public class Storage {
+    private String filePath;
+
+    /**
+     * Constructor for a Storage.
+     *
+     * @param filePath The relative string file path of the saved data.
+     * */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Checks if there is a local saved txt file at the filePath.
+     * If there exists no such .txt file, it will create a new save file at the filePath.
+     * */
+    public void createLocalStorage() {
+        try {
+            File file = new File(filePath);
+
+            if (!file.getParentFile().exists()) {
+                file.getParentFile().mkdirs();
+            }
+
+            if (file.createNewFile()) {
+                System.out.println("I have made a new save file for ya : " + file.getAbsolutePath());
+            } else {
+                System.out.println("You seem to have an existing save file, YAY : " + file.getAbsolutePath());
+            }
+
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    /**
+     * Reads data of commands and adds them to the given TaskList.
+     *
+     * @param data String of commands.
+     * @param tasks TaskList.
+     * @throws TobiasException If the given data contains corrupted commands.
+     * */
+    public static void localToList(String data, TaskList tasks) throws TobiasException {
+        if (data.startsWith("T")) {
+            boolean isDone = Integer.parseInt(data.substring(8,9)) == 1;
+
+            int desc = data.indexOf("|desc");
+            String description = data.substring(desc+5);
```

Comment:
As per the [Coding Standard](https://se-education.org/guides/conventions/java/intermediate.html) there should be a space between operators.
```suggestion
            String description = data.substring(desc + 5);
```